lib/menu.rb

```diff
@@ -0,0 +1,12 @@
+class Menu
+  attr_reader :dishes
+  
+  def initialize
+    @dishes = {"Salted Chilli Ribs" => 5.10,
```

Comment:
You should pass dishes so that the menu can be changed. See the link below: 
https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md#open-closed-principle